scapy/layers/http.py

```diff
@@ -551,8 +551,22 @@ def tcp_reassemble(cls, data, metadata):
                 # It's not Content-Length based. It could be chunked
                 encodings = http_packet[HTTP].payload._get_encodings()
                 chunked = ("chunked" in encodings)
+                is_response = isinstance(http_packet.payload, HTTPResponse)
                 if chunked:
                     detect_end = lambda dat: dat.endswith(b"\r\n\r\n")
+                # HTTP Requests that do not have any content,
+                # end with a double CRLF
+                elif isinstance(http_packet.payload, HTTPRequest):
+                    detect_end = lambda dat: dat.endswith(b"\r\n\r\n")
+                    # In case we are handling a HTTP Request,
+                    # we want to continue assesing the data,
```

Comment:
s/assesing/assessing/

Reply:
Hey, will do. I'll get to it tomorrow :)